training/algorithms.ipynb

```diff
@@ -0,0 +1,192 @@
+{
```

Comment:
<p>General question, I assume the lowercase/underscores are for the nested directory names? Do you think that is preferred over writing the normal formatting of each algorithm? (ie <code>xgboost</code> versus <code>XGBoost</code> , etc)   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3452/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Not sure actually, I was just copying what RTD had. I don't see any problem with changing it to normal formatting - I think that would be preferred TBH. Will change it now.</p><p> </p>